core/src/execution/mod.rs

```diff
@@ -176,6 +177,11 @@ impl<N: NetworkSpec, R: ExecutionRpc<N>> ExecutionClient<N, R> {
             .await
     }
 
+    pub async fn get_client_version(&self) -> Result<String> {
+        let helios_version = env::var("CARGO_PKG_VERSION");
```

Comment:
This method fetches the environment variable at runtime, so it works when you compile Helios yourself, but for a shipped Helios binary it will fail. We should be using `std:env!(VAR_NAME)` which is evaluated at compile time.

Reply:
Done!